bin/generate-methods.js

```diff
@@ -15,9 +15,10 @@
 
```

Comment:
Protobuf typings are really really bad, none of the AST stuff is in there. I'd have to use `any` everywhere to make it work, probably not worth it 😞